edgelet/iotedge/src/main.rs

```diff
@@ -270,6 +270,12 @@ fn run() -> Result<(), Error> {
                         .value_name("IOTHUB_HOSTNAME")
                         .help("Sets the hostname of the Azure IoT Hub that this device would connect to. If using manual provisioning, this does not need to be specified.")
                         .takes_value(true),
+                ).arg(
+                    Arg::with_name("verbose")
```

Comment:
Is there a reason the user would ever not want verbose output?

Reply:
Talked about it with Venkat. I don't think it helps much, but it's consistent with other things.

Reply:
venkat asked for it as a flag. talked to him, it might get changed to --quiet or just removed